src/pairwise.jl

```diff
@@ -149,14 +160,14 @@ function promote_type_union(::Type{T}) where T
 end
 
 function _pairwise(::Val{skipmissing}, f, x, y, symmetric::Bool) where {skipmissing}
-    x′ = x isa Union{AbstractArray, Tuple, NamedTuple} ? x : collect(x)
-    y′ = y isa Union{AbstractArray, Tuple, NamedTuple} ? y : collect(y)
+    x′ = x isa Union{AbstractArray,Tuple,NamedTuple} ? x : collect(x)
+    y′ = y isa Union{AbstractArray,Tuple,NamedTuple} ? y : collect(y)
     m = length(x′)
     n = length(y′)
 
-    T = Core.Compiler.return_type(f, Tuple{eltype(x′), eltype(y′)})
-    Tsm = Core.Compiler.return_type((x, y) -> f(disallowmissing(x), disallowmissing(y)),
-                                     Tuple{eltype(x′), eltype(y′)})
+    T = Core.Compiler.return_type(f, Tuple{eltype(x′),eltype(y′)})
+    Tsm = Core.Compiler.return_type((x, y) -> f(handle_pairwise(x, y)...),
+        Tuple{eltype(x′),eltype(y′)})
```

Comment:
```suggestion
                                     Tuple{eltype(x′),eltype(y′)})
```

Reply:
Again, this is a result of my enthusiasm for the VSCode Format Document command.